istio-configuration/configure-istio.sh

```diff
@@ -31,9 +31,13 @@ spec:
   - host: $INGRESS_IP.nip.io
     http:
       paths:
-      - backend:
-          serviceName: api-gateway-nginx
-          servicePort: 80
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: api-gateway-nginx
+            port:
+              number: 80
```

Comment:
[nitpick] if possible, please use a port name instead of a number. the port name should be defined in the `api-gateay-nginx` service

Reply:
good hint, let's consider changing this in the next iteration